src/core/components/SlQueryBuilder/SlQueryBuilder.css

```
@@ -0,0 +1,38 @@
+@import url('smart-webcomponents-react/source/styles/components/smart.base.css');
+@import url('smart-webcomponents-react/source/styles/components/smart.common.css');
+@import url('smart-webcomponents-react/source/styles/components/smart.querybuilder.css');
+@import url('@ni/nimble-tokens/dist/styledictionary/css/variables.css');
+
+.sl-query-builder .smart-query-builder {
+    /* Tokens from nimble tokens */
+    --smart-primary: var(--ni-nimble-base-digital-green-light);
+
+    /* Tokens from grafana */
+    --smart-background: var(--ni-grafana-background-canvas);
+    --smart-border: var(--ni-grafana-border-medium);
+    --smart-background-color: var(--ni-grafana-text-primary);
+    --smart-ui-state-focus: var(--ni-grafana-focus-color);
+    --smart-border-top-right-radius: var(--ni-grafana-border-radius-default);
+    --smart-border-top-left-radius: var(--ni-grafana-border-radius-default);
+    --smart-border-bottom-right-radius: var(--ni-grafana-border-radius-default);
+    --smart-border-bottom-left-radius: var(--ni-grafana-border-radius-default);
+
+    --smart-ui-state-color-focus: var(--smart-background-color);
+}
+
+.sl-query-builder, .smart-scroll-bar {
```

Are the differences between these selectors intentional?

`.sl-query-builder .smart-query-builder` i.e. `.smart-query-builder` is a child of `.sl-query-builder` in the page

vs

Apply all these styles to both `.sl-query-builder` and `.smart-scroll-bar` separately.

Seems unexpected, is the scrollbar not a child of the query builder? Should this be:

```suggestion
.sl-query-builder .smart-scroll-bar {
```